### PR TITLE
change stuff here and there

### PR DESCRIPTION
**Pull request by nav-github01001**
## Description
Fix fatal flaw in setup.cfg + update readme.md+ change sectins a little

## Checklist

- [ ] - All code changes have been tested, if any
- [ ] - The documentation has been updated accordingly for any code changes
- [ ] - This PR fixes an already existing issue
- [x] - This PR adds a new feature (new functions, methods, classes, parameters, etc.)
- [ ] - This PR fixes a bug
- [ ] - This PR is a breaking change (removes/renames functions, parameters etc.)
- [ ] - This PR is NOT a code change (changes Documentation files, README.md, CONTRIBUTING.md etc.)

### CODE REVIEW

File: EpikCord/__init__.py [hunk 1]
```diff
@@ -1722,6 +1730,22 @@ def register_slash_command(func):
             }
         return register_slash_command
 
+    def get_commands(self) -> List[Union[ClientSlashCommand,ClientUserCommand,ClientMessageCommand]]:
+
+        return [c for c in self.commands]
+
+    def command_unload(self, name:str):
```
Comment (TheUntraceable) [inline EpikCord/__init__.py:1737]: This makes no sense

File: README.md [hunk 2]
```diff
@@ -2,7 +2,15 @@
     <img alt="EpikCord logo" src="docs/EpikCord_Logo.png" width="200"> <!-- Yes I sourced this from DiscordGO because their design inspired me :D-->
 </p>
 
-# EpikCord.py
+
+![Documentation Status](https://readthedocs.org/projects/epikcordpy/badge/?version=latest&style=for-the-badge)
+![PyPi version](https://img.shields.io/badge/pypi-0.4.11-blue?style=for-the-badge)
```
Comment (TheUntraceable) [inline README.md:7]: I don't want to change this each release.

File: EpikCord/__init__.py (hunk 1 above)
Comment (nav-github01001) [reply EpikCord/__init__.py:1737]: what? there are two funcs

File: README.md (hunk 2 above)
Comment (nav-github01001) [reply README.md:7]: Well 1 sec

File: EpikCord/__init__.py (hunk 1 above)
Comment (TheUntraceable) [reply EpikCord/__init__.py:1737]: How do you plan on loading the command back in once you pop it? Why are you raising a `SectionNotFound` error when the command is not valid?

Comment (nav-github01001) [reply EpikCord/__init__.py:1737]: easy
put it in a separate dict
and put it back in reload function

File: README.md (hunk 2 above)
Comment (nav-github01001) [reply README.md]: i turned it off

File: EpikCord/__init__.py (hunk 1 above)
Comment (TheUntraceable) [reply EpikCord/__init__.py:1850]: Just add a `Command.disabled` attribute and then you can do `Command.disable/enable()`? Probably more memory efficient as you are just setting an attribute in contrast to having to having an entirely separate `dict` filled with `Command`s

File: EpikCord/__init__.py [hunk 3]
```diff
@@ -619,6 +619,118 @@ def heartbeater():
         except KeyError:
             return
 
+class Intents:
```
Comment (TheUntraceable) [inline EpikCord/__init__.py:622]: Why did you add the Intents class again? Or did you delete it and move it up here?

**Comment by Jerry-py** [comment]
Fix checks please

File: EpikCord/__init__.py (hunk 3 above)
Comment (nav-github01001) [reply EpikCord/__init__.py:622]: i delete and moved it up

**Comment by nav-github01001** [comment]
> Fix checks please

nothing to worry abt its codespell
